pages/common/wget.md

```diff
@@ -5,24 +5,28 @@
 
 - Download the contents of an URL to a file:
 
-`wget -O {{filename}} {{url}}`
+`wget {{https://example.com}}`
 
-- Limit download speed:
+- Download a single web page and all its associated resources:
 
-`wget --limit-rate={{200k}} {{url}}`
+`wget --page-requisites --convert-links {{https://example.com/somepage.html}}`
```

Comment:
What does `--convert-links` do ?

Reply:
It rewrites links present in the content of the page (in this case links to the scripts, styles, images, etc.) to point to the local versions.

Reply:
Ok